chargeOclock/Source/Other/NetworkAPI.swift

```diff
@@ -16,6 +16,9 @@ extension NetworkAPI {
 		performGetRequest(to: "clients", completion: completion)
 	}
 
+	func removeClient(with id: Int, completion: @escaping NetworkCompletion) {
```

Comment:
also, the method should accept a Client object, and read the id from there...
this way we gain swifts built in type safety...